### ISSUE TITLE
[Question]: Use non-solo Cameras (e.g. E330 pro) without Homebase

### ISSUE THREAD

**Issue by seb2010** [bropat/ioBroker.eufy-security#466]
### Ask your question

Hi guys,
I realized after I bought the camera, that the eufy E330 professional is only usable together with a Homebase. As I do want to connect it to my own NVR system, I am wondering if I could still integrate the camera with this library and get the rtsp-stream going with it.
KR
Seb

**Comment by PhilippEngler** [comment]
No, that's not possible. The underlying client will not simulate a home base, it use the API to communicate/control the devices/home bases.

If you want to use the E330 pro, you need a home base. Only the solo-devices can be operated without home base.

**Comment by seb2010** [comment]
Appearently, the newest firmware of the E330 can make it usable as a solo device, but without a HB3, I cannot upgrade the firmware. Was hoping for some HB3 emulation or other direct access to the E330 to do that.
I was about to crack the micro-USB port open to pla with it, but since I most probably want to return the device, this would not have been a good idea. Unless there are already howtos available.

**Comment by PhilippEngler** [comment]
That is the first time I hear from such firmware updates.

I am not sure if you mean the so called „Multi-Bridge Function“? If so, this function will not solve your problem, the E330 pros _can_ connect to your HomeBase over your local (W)LAN, but you need a HomeBase.

May I ask for a link where you have found information about these firmware update?

**Comment by seb2010** [comment]
https://community.eufy.com/t/e330-without-homebase/4598402/9

**Comment by PhilippEngler** [comment]
Thank you. This is not described in the FAQ nor in the user manuals and I am not sure if the firmware is available. Try to return the cam and get one that is compatible with your setup.
